pkg/cover/cover.go

```diff
@@ -48,3 +60,111 @@ func (cov Cover) Serialize() []uint32 {
 	}
 	return res
 }
+
+const EXT string = ".ko"
+
+func searchModuleName(data []byte) (string, error) {
+	key := []byte("name=")
+	pos := bytes.Index(data, key)
+	if pos == -1 {
+		return "", fmt.Errorf("not find name= keyword")
+	}
+	end := bytes.Index(data[pos:], []byte{0x00})
+	if end == -1 {
+		return "", fmt.Errorf("not find null char")
```

Comment:
We always throw the error away, so these error strings are not really used in any way. It would lead to simpler code and the same semantics to just return empty name here.